config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java

```diff
@@ -122,6 +122,8 @@
 
         metrics.add(new Metric("athenz-tenant-cert.expiry.seconds.last", "athenz-tenant-cert.expiry.seconds"));
 
+        metrics.add(new Metric("jdisc.http.request.prematurely_closed.rate"));
```

Comment:
Please move the `http.status.xxx.rate` metrics here as well, as they are related. I put them in getQrserverMetrics by mistake.

Reply:
Fixed - PTAL